### ISSUE TITLE
incorrectly labeled adducts

### ISSUE THREAD

**Issue by abefrandsen** [daniellyz/MergeION2#10]
I'm finding large errors between the reported precursor m/z and the theoretical precursor m/z for records where the adduct is labeled [M+Na]+, [M+NH4]+, and [M+K]+. In all the cases I checked (and I would hypothesize this is true for every such case), it appears that the precursor m/z would be correctly explained instead by [M+H]+. The source of my data is: `drug_plus.mgf` as https://zenodo.org/record/7531170. A particular case is as follows:
```
BEGIN IONS
PEPMASS=359.1635
RT=N/A
IONMODE=Positive
ADDUCT=[M+Na]+
CHARGE=1
ID=DHSUYTOATWAVLW
SMILES=CC1(C[C@@H]1/C(=N/C(=C\CCCCSC[C@@H](C(=O)O)N)/C(=O)O)/O)C
FORMULA=C16H26N2O5S
FILENAME=NA
DrugBank.ID=NA
INCHIKEY=DHSUYTOATWAVLW-HPTVMLIKSA-N
CPD=NA
Status=NA
CAS=NA
NAME=Massbank:AU212932%%CCMSLIB00005776979
MSLEVEL=2
TIC=3396
```
(the rest of the record is omitted). Note that the [exact mass of the compound](https://pubchem.ncbi.nlm.nih.gov/compound/6602278#section=Computed-Properties&fullscreen=true) here is 358.15624311, so [M+H]+ would explain the given precursor m/z, whereas [M+Na]+ would be much too large.

**Comment by daniellyz** [comment]
Dear Aabefrandsen,
Thank you for reporting this issue!

When generating the database, we purposely merged different spectral records of the same compound - some of these experimental spectra could be reported as its [M+Na]+, [M+NH4]+, or [M+K]+ adduct in their original repository. After merging together, we recalculated the theoretical precursor mass [M+H]+ for the unique record to make the metadata cleaner and more conistent (since these adducts usually have not impact on MS/MS spectrum). 

In summary, [M+Na]+ labelling you saw in this example comes from the original spectral record from public domain, while PEPMASS is the theoretical [M+H]+ mass.

Please let me know if you have any more questions or suggestions.

Best regards
the MergeION team

**Comment by abefrandsen** [comment]
Thanks for answering. 
I would think the errant adduct labels should be adjusted to reflect the actual precursor mass associated with the record in this database.

Can you explain the merging process a bit more? Wouldn't these different adducts indeed have an impact on the MS/MS spectrum, for example would we not expect to see peaks associated with the precursor ion?
